packages/runtime/container-runtime-definitions/src/snapshot.ts

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+import { channelsTreeName } from "@fluidframework/runtime-definitions";
+
+export type ContainerRuntimeSnapshotFormatVersion =
+    /**
+     * Format version is missing from snapshot.
+     * This indicates it is an older version.
+     */
+    | undefined
+    /**
+     * Introduces .metadata blob and .channels trees for isolation of
+     * data store trees from container-level objects.
+     */
+    | "0.1";
+
+export type DataStoreSnapshotFormatVersion =
+    /**
+     * Format version is missing from snapshot.
+     * This indicates it is an older version.
+     */
+    | undefined
+    /**
+     * From this version the pkg within the data store
+     * attributes blob is a JSON array rather than a string.
+     */
+    | "0.1"
+    /**
+     * Introduces .channels trees for isolation of
+     * channel trees from data store objects.
+     */
+    | "0.2";
```

Comment:
i'm wonder if we should start using a number here

Reply:
I was actually thinking the same thing... It gives us out-of-the-box ordering. I think I'll make the change, and just treat "0.1" as 1.